packages/typespec-ts/src/modular/emitSamples.ts

```diff
@@ -513,3 +536,17 @@ function escapeSpecialCharToSpace(str: string) {
   }
   return str.replace(/_|,|\.|\(|\)|'s |\[|\]/g, " ").replace(/\//g, " Or ");
 }
+
+function getEnvironmentVariableName(paramName: string): string {
```

Comment:
Please add client prefix before the env names, e.g TestingClient -> TESTING_ENDPOINT; 
WidgetManagerClient -> WIDGET_MANAGER_ENDPOINT.